docker/ci/Dockerfile

```
@@ -4,7 +4,7 @@ WORKDIR /starcoin
 COPY rust-toolchain.toml /starcoin/rust-toolchain.toml
 COPY scripts/dev_setup.sh /starcoin/scripts/dev_setup.sh
 
-RUN /starcoin/scripts/dev_setup.sh -b -t -y -o -p && \
+RUN /starcoin/scripts/dev_setup.sh -b -t -p && \
```

Hadolint (2.14.0)

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)